test_scripts/unit_tests/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:18.04
+SHELL ["/bin/bash", "-c"]
```

Comment:
Did you mean this? It's the shell for RUN, not the entry point.

Reply:
Oh.. hah.. thought i had removed that.